Server/index.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+const axios = require('axios')
+const cookieParser = require('cookie-parser')
+const app = express()
+require('dotenv').config()
+
+app.use(function (request, response, next) {
+    response.header("Access-Control-Allow-Origin", "http://localhost:9020");
+    response.header("Access-Control-Allow-Credentials", true);
+    next();
+});
+
+const USER = process.env.USER
+const PASSWORD = process.env.PASSWORD
+const PORT = process.env.PORT || 3001;
+const AUTH_ADDRESS = process.env.AUTH_ADDRESS
+const EXPIRATION_TIME = process.env.EXPIRATION_TIME
+const SPARE_TOKEN = process.env.TOKEN
```

Comment:
This token will be expired soon enough. Can you elaborate how are you going to refresh it?

Reply:
Due to a bad if condition my code was requesting new token over and over and I have run out of tokens. I am currently using a hardcoded non-expiring token. Sorry about that.